backend.native/tests/interop/objc/tests/workerAutoreleasePool.kt

```diff
@@ -0,0 +1,153 @@
+import kotlin.native.concurrent.*
+import kotlinx.cinterop.*
+import kotlin.test.*
+import objcTests.*
+
+@Test
+fun testExecute1() =
```

Comment:
I think names like `testExecuteInMainPark` will look better in test failures report.

Reply:
Done.